crypto/rsa/rsa_local.h

```diff
@@ -48,12 +48,12 @@ struct rsa_st {
     BIGNUM *dmp1;
     BIGNUM *dmq1;
     BIGNUM *iqmp;
+    RSA_PSS_PARAMS *pss;
```

Comment:
Why did you leave the comment behind?

Reply:
Oops..

Reply:
You misunderstood.  The comment on line 55 should be moved up.
(but yeah, removing the TODO was a good move nonetheless)

Reply:
You moved one comment too much.  The "be careful..." one belongs with `ex_data`

Reply:
Ok moved it to the correct line (which is not where it was to start with) :)